### DISCUSSION TITLE
Export/Import Customization

### DISCUSSION THREAD

**Discussion by azzamsa** [oae/gnome-shell-pano#326] [Ideas]
Hi,

Is there a way to export or import style customizations? I’ve spent a lot of time customizing the styles, and I'm worried they might get overwritten during an upgrade. I’d like to back them up to a config file (like `css` or `ini`) so I can restore them if needed.

Also, where are these custom styles saved? I couldn’t find them in `~/.local/share/pano@elhan.io/stylesheet.css`.

Thanks!

![image](https://github.com/user-attachments/assets/44ed2572-4723-4efe-8398-7b8ea6efc238)

**Comment by Totto16** [comment]
Currently there is no easy way to import / export them 😓  They should however not be overwritten by an update.

We store them in dconf, as the gnome eco system does by convention, it's not that easy to backup as far as I know, but maybe there exist some tools, but you can look at them by using something like `dconf-editor` , there you go to `/org/gnome/shell/extensions/pano/` and then you can see all the settings, that you made.

![image](https://github.com/user-attachments/assets/8c620029-ff37-47d3-b934-e55af3e1881e)

(Sorry that the screenshot is in german, but just for you to have an idea on where you need to input the settings path)

**Reply by azzamsa** [reply]
Thanks a ton.

`dconf dump /org/gnome/shell/extensions/pano/ > settings.dconf` will ease my distro hopping.
